layouts/partials/custom_footer.html

```diff
@@ -0,0 +1,3 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.20.0/prism.min.js"></script>
```

Comment:
Are you able to leave this bit out of the PR please as this is custom to how you are using it?